onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -388,12 +388,11 @@ ORT_API_STATUS_IMPL(OrtApis::RegisterCustomOpsLibrary, _Inout_ OrtSessionOptions
 }
 
 namespace {
-template <typename Loader>
-OrtStatus* CreateSessionImpl(_In_ const OrtEnv* env, _In_ const OrtSessionOptions* options,
-                             Loader loader, _Outptr_ OrtSession** out) {
+OrtStatus* CreateSessionImpl(_In_ const OrtEnv* /*env*/, _In_ const OrtSessionOptions* options,
```

Comment:
>CreateSessionImpl [](start = 11, length = 17)

'CreateSessionImpl' doesn't seem like the best name if InferenceSession is provided as an input.

Reply:
Changed it to `LoadAndInitializeSession()` to be more indicative of the intent. Does that work ?